core/src/banking_stage.rs

```diff
@@ -1076,16 +1076,29 @@ impl BankingStage {
 
                 if accepting_packets {
                     let (
-                        (sanitized_transactions, transaction_to_packet_indexes),
+                        (
+                            sanitized_transactions,
+                            transaction_to_packet_indexes,
+                            sanitization_error_metrics,
+                        ),
                         packet_conversion_time,
-                    ): ((Vec<SanitizedTransaction>, Vec<usize>), _) = measure!(
+                    ) = measure!(
                         Self::sanitize_unforwarded_packets(
                             buffered_packet_batches,
                             &packets_to_process,
                             bank,
                         ),
                         "sanitize_packet",
                     );
+                    saturating_add_assign!(
+                        dropped_tx_before_forwarding_count,
+                        sanitization_error_metrics.too_many_account_locks
+                    );
+                    saturating_add_assign!(
+                        dropped_tx_before_forwarding_count,
+                        sanitization_error_metrics.account_loaded_twice
+                    );
```

Comment:
Yikes, the "banking_stage-dropped_tx_before_forwarding" metric is incremented in 4 (!!) different places. By incrementing here, you're double counting these errors in that metrics because they are already reported in `sanitize_unforwarded_packets`